releasenotes/notes/Add-Group-exec-perm-for-the-SecretBackend-command-a7fbe1ece1fad50b.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I guess we should remove this file this the other release note include it.

Reply:
One is for the "agent", the other one is for the "DCA". 

Since the feature was design for the DCA, maybe we can remove it from the Agent release notes. Is that ok? event if the option is visible in the agent config?

Reply:
I missed the difference. Let's keep both then